### DISCUSSION TITLE
Root-level pages

### DISCUSSION THREAD

**Discussion by dylanized** [outline/outline#5989] [Ideas]
As a user, I would like to the ability to create pages at the root level.  (Or to move pages to the root level)

In my case, this would be used for "very important" pages that are visited most frequently.

Thanks!

**Comment by Limezy** [comment]
How would this be different from pages that you would tag as favorites ?

**Reply by tommoor** [reply]
I suppose in that the admin could decide. Generally I've tried to avoid enabling pushing content permanently onto users, frequently visited pages are already surfaced in recent after all

**Reply by dylanized** [reply]
Thanks for the followups!  As Tom suggested, it's not necessarily "most frequent" content I'm trying to move to the root, rather it's the content that I just think belongs there.

As for pushing content on people, for one of my projects I'm the only user, so that's not a concern for me. I just would like to customize the table of contents to match the mental model in my head.   Thanks for listening!

**Reply by tommoor** [reply]
We have "Pin to Home" which you will probably find useful if you haven't seen that, I'm not against a "Pin to Sidebar", or similar but honestly we need more engineering bandwidth at the moment to tackle such things.
